src/main.rs

```diff
@@ -216,6 +220,12 @@ fn settings() -> Result<Settings, Failure> {
                 .long(REPO_OPTION)
                 .help("Sets the Docker repository"),
         )
+        .arg(
+            Arg::with_name(DOCKER_CLI_OPTION)
+                .value_name("CLI")
+                .long(DOCKER_CLI_OPTION)
+                .help("Sets the Docker CLI command"),
```

Comment:
Perhaps this should say `Sets the Docker CLI binary` (instead of `command`) or just `Sets the Docker CLI`, since it's just the executable and doesn't include any arguments.

(It's still kind of awkward, but I can't really think of a perfect description for this.)